install/kubernetes/cilium/templates/hubble-relay-configmap.yaml

```
@@ -0,0 +1,29 @@
+{{- if .Values.hubble.relay.enabled }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hubble-relay-config
+  namespace: {{ .Release.Namespace }}
+data:
+  config.yaml: |
+    peer-service: unix://{{ .Values.hubble.socketPath }}
+    listen-address: {{ .Values.hubble.relay.listenHost }}:{{ .Values.hubble.relay.listenPort }}
+    dial-timeout: {{ .Values.hubble.relay.dialTimeout }}
+    retry-timeout: {{ .Values.hubble.relay.retryTimeout }}
+    sort-buffer-len-max: {{ .Values.hubble.relay.sortBufferLenMax }}
+    sort-buffer-drain-timeout: {{ .Values.hubble.relay.sortBufferDrainTimeout }}
+{{- if .Values.hubble.relay.tls.enabled }}
```

When Hubble is configured with mTLS, we want relay to be configured accordingly.
```suggestion
{{- if .Values.hubble.tls.enabled }}
```

---

fixed